core/locale/domain/dateTime.go

```diff
@@ -17,6 +23,27 @@ func (dtf *DateTimeFormatter) SetDateTime(time time.Time, localtime time.Time) {
 	dtf.localDateTime = localtime
 }
 
+// SetLocation sets location for local time
+func (dtf *DateTimeFormatter) SetLocation(loc string) *DateTimeFormatter {
+	location, err := time.LoadLocation(loc)
+
+	if err != nil && dtf.logger != nil {
```

Comment:
Condition needs to be split, if there is an error and no logger the invalid location is still used which leads to a panic

Reply:
You are right. It has been fixed.